content/en/continuous_integration/setup_pipelines/buildkite.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Set up Tracing on a Buildkite Pipeline
+kind: documentation
+further_reading:
+    - link: "/continuous_integration/explore_pipelines"
+      tag: "Documentation"
+      text: "Explore Pipeline Execution Results and Performance"
+    - link: "/continuous_integration/troubleshooting/"
+      tag: "Documentation"
+      text: "Troubleshooting CI"
+---
+
+{{< site-region region="us,eu" >}}
+## Configuring the Datadog integration
+
+The Datadog integration for [Buildkite][2] works by using [webhooks][1] to send pipeline data to Datadog.
+
+Go to **Settings > Notification Services** in Buildkite and add a new webhook:
+
+{{< site-region region="us" >}}
+* **Webhook URL**: `https://webhooks-http-intake.logs.datadoghq.com/api/v2/webhook/?dd-api-key=<API_KEY>` where `<API_KEY>` is [your Datadog API key][1].
+* **Events**: Select `job.finished` and `build.finished`.
+* **Pipelines**: Select all pipelines or the subset of pipelines you want to trace.
+
+[1]: https://app.datadoghq.com/account/settings#api
+{{< /site-region >}}
+
+{{< site-region region="eu" >}}
+* **Webhook URL**: `https://webhooks-http-intake.logs.datadoghq.eu/api/v2/webhook/?dd-api-key=<API_KEY>` where `<API_KEY>` is [your Datadog API key][1].
+* **Events**: Select `job.finished` and `build.finished`.
+* **Pipelines**: Select all pipelines or the subset of pipelines you want to trace.
+
+[1]: https://app.datadoghq.eu/account/settings#api
```

Comment:
Shouldn't this be https://app.datadoghq.com/organization-settings/api-keys ?

Reply:
You are right, I'll change it, but we also have to change it in the GitLab docs (I copied the URL from there), I'll open another ticket and update that one as well.